src/varsha/OccurenceOfCharacter.java

```diff
@@ -0,0 +1,35 @@
+package varsha;
+
+//Print only characters occurring exactly twice:
+//input: teecccchhhnno;
+//output: en"
+
+class OccurenceOfCharacter {
+
+	int getCountOfChar(String input, char character) {
+		int count = 0;
+		for (int i = 0; i < input.length(); i++) {
+			char ch = input.charAt(i);
+			if (ch == character) {
+				count++;
+			}
+		}
+		return count;
+	}
+
+	void printCharacterHavingCount2(String input) {
+		for (int i = 0; i < input.length(); i++) {
+			char ch = input.charAt(i);
+			int Count = getCountOfChar(input, ch);
```

Comment:
Always start variable name with lowercase

Reply:
yes, By mistake it capital otherwise i will keep same letter only .
next time i will take this in consideration. sorry
Thank you